cypress/fixtures/fireboltCalls/localization.json

```diff
@@ -102,7 +102,7 @@
     "PH_FOR_LOCALIZATION_COUNTRYCODE": {
         "method": "localization.countryCode",  
         "validationJsonPath": "result",
-        "content": "LOCALIZATION_COUNTRYCODE"  
+        "content": "LOCALIZATION_COUNTRYCODE_PH"  
     },
```

Comment:
dont understand the change here. Which test case will  be fixed with this change

Reply:
Updated localization regEx validation to content validation. So, both method and event have same validation.
Reports:
Localization: [Multiple Cucumber HTML Reporter (comcast.com)](https://reporting.firecert.comcast.com/cucumberReports/dfb3b774-4121-4c31-b5fd-cf7c92101a0c/index.html)
LocalizationManage: [Multiple Cucumber HTML Reporter (comcast.com)](https://reporting.firecert.comcast.com/cucumberReports/59cfe590-e588-45dc-815e-295206326e23/index.html)